template/experiment-server/webserver.py

```diff
@@ -5,14 +5,15 @@
 from fastapi import FastAPI, Request
 from fastapi.staticfiles import StaticFiles
 from fastapi.responses import JSONResponse, FileResponse, RedirectResponse
+from starlette.middleware.sessions import SessionMiddleware
 
 app = FastAPI()
+app.add_middleware(SessionMiddleware, secret_key="W<(nwGTe8I9MX$UHftcK")
```

Comment:
This secret probably shouldn't be checked in - I'm surprised that it wasn't caught by GitHub.